source/GameInterface/Services/Clans/Messages/ClanKingdomChange.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Common.Messaging;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.Actions;
+
+namespace GameInterface.Services.Clans.Messages
+{
+    /// <summary>
+    /// Local event when a clan kingdom is changed from game interface
+    /// </summary>
+    public record ClanKingdomChange : IEvent
+    {
+        public string ClanId { get; }
+        public string NewKingdomId { get; }
+        public int Detail { get; }
+        public int AwardMultiplier { get; }
+        public bool ByRebellion { get; }
+        public bool ShowNotification { get; }
+
+        public ClanKingdomChange(string clanId, string newKingdom, int detail, int awardMultiplier, bool byRebellion, bool showNotification)
```

Comment:
These are IEvents from the GameInterface, and should end with a past-tense verb. In this case `ClanKingdomChanged` but this name is already used. Update the other name to follow the pattern `ChangeClanKingdom` and be an ICommand.

Do this for all events like this